extensions/amp-subscriptions/0.1/amp-subscriptions.js

```diff
@@ -119,13 +116,19 @@ export class SubscriptionService {
    * This method registers an auto initialized subcription platform with this service.
    *
    * @param {string} serviceId
-   * @param {!SubscriptionPlatform} subscriptionPlatform
+   * @param {function(!ServiceConfig, !PageConfig):Promise<!SubscriptionPlatform>} subscriptionPlatformFactory
```

Comment:
I'm confused. The `ServiceConfig` is a list of platform configs. Instead this should be a platform config, which is a free-form object, right?